### PR TITLE
Added the ML model for disease prediction

### PR DESCRIPTION
**Pull request by BhoomiAgrawal12**
# Fixes Issue🛠️
#5 

Closes #

# Description👨‍💻
I have made an ML model that predicts the diseases based on the symptoms added and diplays a probability for the happening of the disease,description about the disease and the precautions to be taken for the same.These model involves the usage of four different datasets-symptoms and disease relation,precaution,severity of disease and description  of disease from kaggle,I have compared different ML models for classification namely Random Forest, XGBoost, LightGBM, GradientBoost and ExtraTrees and on the basis of accuracy and precision score ExtraTrees is the best.However integration of the ml model with the website is still is not done but I have added the pickel files in the backend folder.As integration takes time  but I am ready to work on it as well.

# Type of Change📄
- [✔ ] New feature (non-breaking change which adds functionality)

# Checklist✅

- [ ✔] I am an Open Source contributor
- [✔ ] I have performed a self-review of my code
- [✔ ] My code follows the style guidelines of this project
- [✔ ] I have commented on my code, particularly in hard-to-understand areas

# Screenshots/GIF📷
Screenshot 2025-01-17 231556
Screenshot 2025-01-17 231618
Screenshot 2025-01-17 231654

### CODE REVIEW

**Comment by PratikMane0112** [comment]
Hey @BhoomiAgrawal12, thanks for backend. You have lots of follow up on this now, 
- Integrating model with website
- Its UI
- Make README.md about explaining model (its dataset , no of coloums, rows, features, algo)
- Also Guidance for ML Enthusiast contributors about model 
- You can create new issue & PR everytime & I will add advance label to each
Happy Coding !!!!

**Comment by PratikMane0112** [comment]
I am unable to assign you points as website is broken currently !!!

Screenshot 2025-01-18 004418

**Comment by BhoomiAgrawal12** [comment]
yaa sure I will be working on the the next steps,till then can you please check for the points once.

**Comment by PratikMane0112** [comment]
> yaa sure I will be working on the the next steps,till then can you please check for the points once.

Done !!
